app/controllers/admin/talentlms_controller.rb

```diff
@@ -43,6 +48,20 @@ def destroy
       @config = config_scope.find(params[:id].to_i)
     end
 
+    private def set_site_config
+      @site_configs = site_config_scope.where(id: 1).first_or_create
```

Comment:
Did you intend to leave `where(id: 1)` here?

Reply:
This is how it's being referenced elsewhere in the app ([example](https://github.com/greenriver/hmis-warehouse/blob/dg/talent-captive-portal_6669/app/controllers/admin/configs_controller.rb#L30), [example](https://github.com/greenriver/hmis-warehouse/blob/dg/talent-captive-portal_6669/app/controllers/admin/de_duplication_controller.rb#L42), [example](https://github.com/greenriver/hmis-warehouse/blob/dg/talent-captive-portal_6669/spec/models/grda_warehouse/system_cohorts/system_cohort_spec.rb#L31)) so I matched to stay consistent. It does feel like removing the `.where(id: 1)` from each of these locations and going with `GrdaWarehouse::Config.first_or_create` would be better. Happy to adjust it in all of these spots if you feel it is worth doing.

Reply:
Oh, I mis-read that, I was thinking `GrdaWarehouse::Config` was some sort of other Talent related config.  Huh, that's weird, but I think the issue is that you have to have something between the class and `.first_or_create`.  Let's leave this as is.  Thanks.